cognite/client/utils/_text.py

```diff
@@ -0,0 +1,39 @@
+import functools
```

Comment:
Nice change, but would be even nicer in a separate PR! The PR is not really reviewable commit-by-commit, so this causes noise when reviewing all files at once.

Reply:
Will move to a separate PR then 😉

Reply:
https://github.com/cognitedata/cognite-sdk-python/pull/1179